.changeset/tame-keys-provide.md

```diff
@@ -0,0 +1,18 @@
+---
+'@primer/react-brand': minor
+---
+
+Use VideoPlayer component to render hosted videos
+
+```jsx
+<VideoPlayer title="HLS On Demand Demo" crossOrigin="anonymous">
```

Comment:
As `crossOrigin` isn't indicated as a required prop, it shouldn't need to be mentioned in the usage example.

```suggestion
<VideoPlayer title="HLS On Demand Demo">
```

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/6f9a9c8740844ac6d50be622f1c9aab106c564dc)